docs/tools/vdb_table/data/aperturedb.json

```diff
@@ -9,51 +9,51 @@
     "slug": "aperturedb"
   },
   "oss": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://hub.docker.com/r/aperturedata/aperturedb-community",
+    "comment": "ApertureDB community edition is free and a great way to test all the database features, locally."
   },
   "license": {
-    "value": "",
-    "source_url": "",
-    "comment": ""
+    "value": "Proprietary",
+    "source_url": "https://github.com/aperture-data",
+    "comment": "ApertureDB code is proprietary but we have open source clients, dataset, and applications repositories."
   },
   "dev_languages": {
     "value": [
-      ""
+      "c++"
     ],
-    "source_url": "",
+    "source_url": "https://github.com/aperture-data",
     "comment": ""
   },
-  "vector_launch_year": 0,
+  "vector_launch_year": 2021,
   "metadata_filter": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/concepts/Schema",
+    "comment": "Complete graph database support alongside vector search"
   },
   "hybrid_search": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/python_sdk/object_wrappers/Descriptors",
+    "comment": "Blending graph filtering with vector search. Easy to introduce search algorithms. You can search across multiple embedding sets as well (collections)"
   },
   "facets": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/query_language/Reference/shared_command_parameters/constraints",
     "comment": ""
   },
   "geo_search": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://docs.aperturedata.io/HowToGuides/start/Embeddings",
+    "comment": "Coordinates can be represented in metadata or as embeddings"
   },
   "multi_vec": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/HowToGuides/Applications/similarity_search#all-of-the-above-and-some-more-can-be-done-by-executing-this-through-adb-cli",
+    "comment": "You can attach multiple embeddings per source of data for multimodal data types in different embeddings collections or within the same (regions of interest or full data embeddings)."
   },
   "sparse_vectors": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://docs.aperturedata.io/query_language/Reference/descriptor_commands/desc_set_commands/AddDescriptorSet",
```

Comment:
I don't see a mention of sparse vectors at the link - can you clarify? For clarity here is the field description:
Bring your own sparse vectors (e.g. TF-IDF) and have them be indexed and available for retrieval in the database.